traffic_ops/traffic_ops_golang/api/api.go

```diff
@@ -60,6 +61,12 @@ func WriteResp(w http.ResponseWriter, r *http.Request, v interface{}) {
 
 // WriteRespRaw acts like WriteResp, but doesn't wrap the object in a `{"response":` object. This should be used to respond with endpoints which don't wrap their response in a "response" object.
 func WriteRespRaw(w http.ResponseWriter, r *http.Request, v interface{}) {
+	if getWroteResp(r) {
```

Comment:
My only suggestion would be renaming these `respWritten` and `setRespWritten`. Overall I think this PR looks good.

Reply:
Done